config/schema/web_page_archive.schema.yml

```diff
@@ -2,6 +2,18 @@ web_page_archive.settings:
   type: config_object
   label: 'Web page archive settings'
   mapping:
+    system:
+      type: mapping
+      mapping:
+        node_path:
+          type: string
+          label: 'NodeJS path'
+        phantomjs_path:
+          type: string
+          label: 'PhantomJS path'
+        chrome_path:
+          type: string
+          label: 'Chrome path'
```

Comment:
just reminding myself to make sure the schema ultimately matches what gets placed in the form as it is inconsistent right now .